examples/tutorials/analysis-time/TimeResolvedSpectroscopyEstimator.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #84.            result = self.fit.run(datasets_in_bin)
</pre><p>The result of the fit is not propagated to higher level? What about is it does not converge?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/5427/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
good point @bkhelifi ! We can have a status message column in the table.
Do we need to save all the fit_results ? How should that be serialized? 

@cgalelli Do we need some measure of fit statistics in every bin? What should that be, del TS with and without the model?

Reply:
Hi @bkhelifi and @AtreyeeS , good question. In general as I wrote below I think it would be a good idea at least to have an option where user could ask for the full serialization of fit_results

Reply:
Indeed, this option could be great.

 And the new (mandatory) column should contain the status... but one should handle:
- what about if migrad converges but not hess (the errors)? maybe the status should be the logical AND (to be documented).
- what about the sherpa and scipy backends?

Reply:
I have seen that... I was speaking about their success flag. They are well defined? Just as a check...